frontend/src/pages/budgetLines/ProjectAgreementSummaryCard.jsx

```diff
@@ -28,7 +28,7 @@ export const ProjectAgreementSummaryCard = ({
                 </div>
                 <div className="margin-left-5">
                     <dt className="margin-0 text-base-dark">Fee Rate</dt>
-                    <dd className="margin-0 text-semibold">{fee && `${fee * 100}%`}</dd>
+                    <dd className="margin-0 text-semibold">{fee && `${fee * 100}`}%</dd>
```

Comment:
We could probably use the `CurrencyFormat` (something we already use) and the `format` to create a standardized "Percentage" format to use across the board 🤔 - not required for this PR, but just a note...

Reply:
good point